src/renderer/columnRenderers/colFileName.tsx

```diff
@@ -29,6 +29,7 @@ export const ColFileName = ( props: IFileNameProps ) =>  {
 			<div
 				className='fileList_File_Name_Row'
 				title={formattedFilePath}
+				data-path={dataArgs.cellData}
```

Comment:
What do you think about binding `imageID's` to each of these and then grabbing the appropriate data from redux? It is just a hunch and I have not tested this, but I'm wondering if the approach of binding the whole `ImageData` object might contribute to scroll performance in long file lists. 🤷  Anyways, might be worth doing some more testing in the future and making that change only if it is necessary in the future.

Reply:
Ya I agree. Regardless of performance, and you may be correct, using redux to get the data to the contextMenu is cleaner.